components/hab/src/command/pkg/binlink.rs

```diff
@@ -38,7 +38,7 @@ pub fn start(
         dst_path.display()
     ))?;
     let pkg_install = PackageInstall::load(&ident, Some(fs_root_path))?;
-    let src = match hfs::find_command_in_pkg(binary, &pkg_install, fs_root_path)? {
+    let src = match hfs::find_command_in_pkg(binary, &pkg_install)? {
```

Comment:
We need to find a command sometimes in a temporary filesystem root (such as when creating Docker images) or in Studio roots. This is why `fs_root_path` is required here.